### PR TITLE
fix test result for goto-cl

### PR DESCRIPTION
**Pull request by kroening**

### CODE REVIEW

File: regression/ansi-c/function_return1/test.desc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 CORE
 main.c
 --verbosity 2
-^main.c:3:1: warning: function has return void but a return statement returning signed int$
+^.*: function has return void but a return statement returning signed int$
```
Comment (tautschnig) [inline regression/ansi-c/function_return1/test.desc]: Nit pick: just remove `^.*`
And just curious: what _does_ the output of goto-cl look like?

**Review by tautschnig — APPROVED** [review]

File: regression/ansi-c/function_return1/test.desc (hunk 1 above)
Comment (kroening) [reply regression/ansi-c/function_return1/test.desc]: file main.c line 3 function fun: function has return void but a return statement returning signed int

Comment (kroening) [reply regression/ansi-c/function_return1/test.desc]: We could eventually imitate the style of the real CL, which is
main.c(3): warning C4098: 'fun': 'void' function returning a value

**Review by smowton — APPROVED** [review]

**Review by thk123 — APPROVED** [review]
Does anyone know why the list of failing tests is duplicated between `appveyor.yml` and `buildspec-windows.yml`?

**Pull request merged**